src/side-nav/side-nav-header.jsx

```diff
@@ -0,0 +1,67 @@
+let React = require('react/addons');
+let StylePropable = require('../mixins/style-propable');
+let MenuItem = require('../lists/list-item');
+
+let SideNavHeader = React.createClass({
+
+  mixins: [StylePropable],
+
+  contextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+
+  propTypes: {
+    disabled: React.PropTypes.bool,
+    lineHeight: React.PropTypes.string,
+    innerDivStyle: React.PropTypes.object,
+    insetChildren: React.PropTypes.bool,
+  },
+
+  getTheme() {
+    if(this.context.muiTheme.component.sideNav)
+      return this.context.muiTheme.component.sideNav;
+    else
+      return {
+        headerItemBackgroundColor: '#2196f3',
+        headerItemTextColor: '#000000',
```

Comment:
Why do you provide here some "default" theme properties? Is there a real case when this.context.muiTheme.component.sideNav is missing?